File: Beecrowd/Python/bee1149.py
usr_input = map(int, input().split())
valid_values = []
total = 0

for value in usr_input:
    if value > 0:
        valid_values.append(value)

a = valid_values[0]
n = valid_values[1]

for i in range(a, a + n):
    total += i 

print(total)

File: Beecrowd/C/bee1149.c
#include <stdio.h>

int main() {
  int a, n, total = 0;

  scanf("%d %d", &a, &n);

  while (n <= 0) {
    scanf("%d", &n);
  }

  for (int i = 0; i < n; i++) {
    total += a + i;
  }

  printf("%d\n", total);

  return 0;
}
